xpublish/rest.py

```diff
@@ -49,6 +56,44 @@ def __init__(self, xarray_obj):
         self._attributes = {}
         self._variables = {}
         self._encoding = {}
+        self._cache_kws = {'available_bytes': 1e6}
+        self._app_kws = {}
+        self._cache = None
+        self._initialized = False
+
+    def __call__(self, cache_kws=None, app_kws=None):
+        """
+        Initialize this RestAccessor by setting optional configuration values
+
+        Parameters
+        ----------
+        cache_kws : dict
+            Dictionary of keyword arguments to be passed to ``cachey.Cache()``
+        app_kws : dict
+            Dictionary of keyword arguments to be passed to
+            ``fastapi.FastAPI()``
+
+        Notes
+        -----
+        This method can only be inoked once.
+        """
+        if self._initialized:
+            raise RuntimeError('This accessor has already been initialized')
+        self._initialized = True
+
+        # update kwargs
+        if cache_kws is not None:
+            self._cache_kws.update(cache_kws)
+        if app_kws is not None:
+            self._app_kws.update(app_kws)
+        return self
```

Comment:
@lsetiawan - please give me your thoughts on this change. I've made `init_app()` a private method and instead exposed the accessor's `__call__` method as the preferred way to set options. So now, you're app configuration would go like this:

```python
ds.rest(app_kws=dict(title='foo')).serve()
```

Reply:
This looks really great! i like the cleanness of this. Nice work!

Reply:
> ds.rest(app_kws=dict(title='foo')).serve()

This is great! I didn't know that this is the proper way to set options when using the accessor until now :)

Reply:
I don't think we have a "proper" way defined but this is fairly sane.

Reply:
This would be great to have in the xarray documentation.

Reply:
+1:  https://github.com/pydata/xarray/issues/3829